app/scripts/web/blueprints/grading.py

```
@@ -0,0 +1,692 @@
+
+    if rubric_path and rubric_path.suffix.lower() == ".json":
+        try:
+            data = json.loads(rubric_path.read_text(encoding="utf-8"))
+            raw  = data.get("criteria", [])
+            rubric_criteria = [
+                {"criterion_id": f"C{i+1}", "criterion_name": c.get("criterion_name", f"C{i+1}"),
+                 "max_points": float(c.get("max_points", 0)), "checklist_items": c.get("checklist_items", [])}
+                for i, c in enumerate(raw) if float(c.get("max_points", 0)) > 0
+            ]
+        except Exception:
+            pass
```

Ruff (0.15.11)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)